pool/pooler.go

```diff
@@ -155,3 +161,9 @@ type Pooler interface {
 	ExecuteAsync(expr string, args interface{},
 		mode Mode) *tarantool.Future
 }
+
+// PoolerTopology is the interface that must be implemented by a connection pool and describe topology methods.
```

Comment:
Linter error here: line is too long. I also think it could be shortened.
```suggestion
// PoolerTopology describes pooler topology methods.
```